puma/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG - puma
 
+## 1.2.1 / 2023-10-05
+
+***Fixed***:
+
+* Update conf.yaml to refect http prefix ([#1887](https://github.com/DataDog/integrations-extras/pull/1887))
```

Comment:
Just saw this typo that I missed on my first pass.

```suggestion
* Update conf.yaml to reflect http prefix ([#1887](https://github.com/DataDog/integrations-extras/pull/1887))
```

Reply:
mb i too didn't realize this.